docs/slack-apps/slack-packages.md

```diff
@@ -0,0 +1,65 @@
+---
+id: slack-packages
+title: Slack Packages
+sidebar_label: Slack Packages
+---
+
+We've built in a variety of functions to make it easier to build Dark apps with Slack. Use these functions from autocomplete (dark/slack/Slack::), the same way you would any other function.
```

Comment:
Can you use backticks around code, such as `dark/slack/Slack::`, types such as `String`, `Dicts`, etc.